### ISSUE TITLE
[Bug] 10.8.7 Incompatible

### ISSUE THREAD

**Issue by DevilsDesigns** [ConfusedPolarBear/intro-skipper#103]
**Describe the bug**

The intro skipper is fully unfunctional in 10.8.7. It works on 10.8.5 then when i upgrade to 10.8.7 it doesnt. Then once i downgrade it works again.

**System information**

- Jellyfin installation method: 

Jellyfin Windows

- OS: 
 Windows 11

**Support bundle**

[go to Dashboard -> Plugins -> Intro Skipper -> Support Bundle (at the bottom of the page) and paste the contents of the textbox here]

**Jellyfin logs**

their are no errors in the logs.

**Comment by ConfusedPolarBear** [comment]
I cannot reproduce this using Jellyfin 10.8.7, installed with the installer, and running on Windows 11 Pro (22H2, build 22621.525). Plugin version 0.1.7 installs correctly and successfully identifies introductions.

> The intro skipper is fully unfunctional in 10.8.7. It works on 10.8.5 then when i upgrade to 10.8.7 it doesnt. Then once i downgrade it works again.

1. What version of the plugin do you have installed?
2. Can you elaborate on what you mean by "fully unfunctional"? Does the plugin:

    * Have a status other than `Active`?
    * Report that the detect introductions task failed?
    * Not display the skip intro button?

**Comment by DevilsDesigns** [comment]
i apologize let me eleborate. I have all intro skipping fully functional on 10.8.5 ie. it skips automatically no issues. once i updated 10.8.7 it doesnt auto skip anymore on any devices. the status on the plugin is active on windows 11 pro jellyfin install. It doesnt show any errors in the log as i can see. It just doesnt run anymore on 10.8.7 even though the plugin is active and running

**Comment by DevilsDesigns** [comment]
yes like i said it works in 10.8.5 same settings same auto skipping and integreated player are all working on web and android and google-tv but once i update to 10.8.7 it broke something to where it doesnt work anymore? I had scanned with 10.8.5 do i need to delete the old scan and rescan all edls again?

**Comment by ConfusedPolarBear** [comment]
> do i need to delete the old scan and rescan all edls again?

No, the auto skip feature doesn't use EDL files.

> I have all intro skipping fully functional on 10.8.5 ie. it skips automatically no issues. once i updated 10.8.7 it doesnt auto skip anymore on any devices.

1. Can you update the initial post to include your support bundle from the 10.8.7 installation?
2. Do you have both `Automatically skip intros` and `Automatically skip intros in the first episode of a season` checked in the plugin's settings?
3. Can you enable verbose (not debug) [logging](https://github.com/ConfusedPolarBear/intro-skipper/blob/master/docs/debug_logs.md#how-to-enable-verbose-logs) and:
    1. Restart Jellyfin
    2. Using the web interface, play an episode that has an introduction which used to automatically skip in 10.8.5
    3. Post the resulting log file

**Comment by roib20** [comment]
Possible duplicate of #128
